applications/callflow/src/module/cf_ring_group.erl

```diff
@@ -134,6 +134,11 @@ start_builder(EndpointId, Member, Call) ->
       end
      ).
 
+-spec member_active(kz_json:object()) -> boolean().
+member_active(Member) ->
+    DisableUntil = kz_json:get_value(<<"disable_until">>, Member, 0),
+    calendar:datetime_to_gregorian_seconds(calendar:universal_time()) > DisableUntil.
```

Comment:
No timezone is taken into account. Should it instead use the callee's timezone?
`disable_until` is currently only used to toggle the user on and off through `log_in_out_ring_group`, however as a potential enhancement it could be used to log a user out temporarily.